lib/trino-parquet/src/main/java/io/trino/parquet/ParquetReaderOptions.java

```diff
@@ -120,127 +120,159 @@ public DataSize getSmallFileThreshold()
 
     public ParquetReaderOptions withIgnoreStatistics(boolean ignoreStatistics)
```

Comment:
Inline and remove all `with..` methods from `ParquetReaderOptions`, in favour of using `with..` methods from just introduced builder.